doc/source/examples/unresolved-cfd-dem/liquid-solid-fluidized-bed/liquid-solid-fluidized-bed.rst

```diff
@@ -0,0 +1,500 @@
+          set Function expression = 0.157033
+        end
+        subsection v
+          set Function expression = 0
+        end
+        subsection w
+          set Function expression = 0
+        end
+      end
+    end
+
+The additional sections for the CFD-DEM simulations are the void fraction subsection and the CFD-DEM subsection. These subsections are described in detail in the `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ .
+
+Void Fraction
+~~~~~~~~~~~~~~~
+
+We choose the `particle centroid method (PCM) <../../../parameters/unresolved-cfd-dem/void-fraction.html>`_ to calculate void fraction. The ``l2 smoothing factor`` we choose is around the square of twice the particle’s diameter, as in the other examples.
+ 
+.. code-block:: text
+
+    subsection void fraction
+      set mode                = pcm
+      set read dem            = true
+      set dem file name       = dem
+      set l2 smoothing factor = 0.000028387584
+      set bound void fraction = false
+    end
+
+.. note::
+    Note that void fraction is not bound in this case. The particles used in this example obligates us to use a very coarse mesh. Bounding void fraction lead to instability in the present case.
+
+CFD-DEM
+~~~~~~~~~~
+
+Different from gas-solid fluidized beds, buoyancy, pressure force, shear stress are not negligible. All these forces are considered in this example.
+
+Saffman lift force is proven to be very important to properly reproduce particles' dynamics in the liquid-fluidized bed `[1] <https://doi.org/10.1016/j.powtec.2023.118652>`_.
+
+.. code-block:: text
+
+    subsection cfd-dem
+      set vans model         = modelA
+      set grad div           = true
+      set drag model         = rong
+      set buoyancy force     = true
+      set shear force        = true
+      set pressure force     = true
+      set saffman lift force = true
+      set coupling frequency = 100
+      set void fraction time derivative = false
+    end
+
+.. warning::
+    Void-fraction time-derivative lead to significant instability in the case of liquid-fluidized beds, hence we do not use it.
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+We use the inexact Newton non-linear solver to minimize the number of time the matrix of the system is assembled. This is used to increase the speed of the simulation, since the matrix assembly requires significant computations.
+
+.. code-block:: text
+
+  subsection non-linear solver
+    set solver           = inexact_newton
+    set tolerance        = 1e-10
+    set max iterations   = 10
+    set verbosity        = verbose
+  end
+
+Linear Solver
+~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection linear solver
+      set method                                = gmres
+      set max iters                             = 5000
+      set relative residual                     = 1e-3
+      set minimum residual                      = 1e-11
+      set ilu preconditioner fill               = 1
```

Comment:
Do you really need a fill level of 1?

Reply:
Nope. I am testing for performance, but it does seem necessary.

Reply:
Performance-wise it was terrible compared to fill level 1. I tested on 32 cores and it took about 3 times as much.